cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerCapacityConfigFileResolver.java

```diff
@@ -178,8 +179,9 @@ public BrokerCapacityInfo capacityForBroker(String rack, String host, int broker
                                         _capacitiesForBrokers.get(DEFAULT_CAPACITY_BROKER_ID).diskCapacityByLogDir(),
                                         _capacitiesForBrokers.get(DEFAULT_CAPACITY_BROKER_ID).numCpuCores());
         } else {
-          throw new TimeoutException(String.format("Unable to resolve capacity of broker %d. Either allow capacity estimation "
-                                                   + "or add broker's capacity information in file %s.", brokerId, _configFile));
+          throw new BrokerCapacityResolvingException(String.format("Unable to resolve capacity of broker %d.Either (1) adding the "
```

Comment:
Inconsistent use of `add` and `adding` 
should we reformat it as -> `Either add (1) XXX, or (2) YYY.`